proxa/Docusign-Integration
# feat: added better react front-end page to interact with the server

Test procedure:

1. gradle bR
2. open http://localhost:8080
3. you *should* see the main page.  there is still some other issue going on though if you check the console....not sure what's happening there.
4. If you click on a button you will be redirected to the authentication portal
5. If you authenticate, you should get a token from Docusign to be used for future API calls

Also, react live development server is set up!
1. cd into src/main/webapp
2. run rpm run start:dev
3. navigate to http://localhost:3000
3. you can now use hot reloading for front-end development (you may have to change the emit setting in the tsconfig.json because every time you build the app with gradle it reverts it for some reason)

Notes:  This still needs work obviously.
